hal/src/b5som/network/network.cpp

```diff
@@ -157,12 +157,16 @@ int if_init_platform_postpone(void*) {
         uint8_t dummy;
         if (!if_get_index(en2->interface(), &dummy)) {
             auto wizNetif = reinterpret_cast<WizNetif*>(en2);
-            if (wizNetif->isPresent()) {
-                return 0;
+            for (uint8_t retries = 0; retries < 10; retries++) {
+                if (wizNetif->isPresent()) {
```

Comment:
Maybe let's move this retry logic into `isPresent()` itself?

Reply:
That will increase the boot time, since `isPresent()` is also invoked when creating the `Wiznetif`. If the auxiliary power is not enabled, those retries make no sense.

Reply:
In that case let's make this an optional parameter for `isPresent(bool retry = false)`